Module4.3_Search_and_Sorting_Algorithms/activities/Leet Code Activity/4.md

```diff
@@ -0,0 +1,41 @@
+Next up is the while loop. It is here that our solution will iterate over each and every element inside the linked list once:
```

Comment:
maybe give a brief reminder of how insertion sort works. that will give a lot more context if people don't remember

Reply:
I already reminded them in the first card. Do you want me to remind them again here?